test/test_ops.py

```diff
@@ -587,6 +587,21 @@ def test_dot(self):
     with self.assertRaises(AssertionError):
       a = Tensor(3.14)
       a.matmul(a)
+  def test_mulacc_with_zero_strides(self):
```

Comment:
i like the old test that uses const that motivates this fix, consider adding that back in addition to this

Reply:
Done.